proxies.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Confirming you intended this file to be added?

Reply:
This file is not needed so I will delete it.